scripts/globals/events/starlight_celebrations.lua

```diff
@@ -612,12 +621,17 @@ end
 
 function xi.events.starlightCelebration.merryMakersGoblinOnTrade(player, npc, trade, id)
     local item = trade:getItemId()
-    if item == 4495 then
-        player:setLocalVar("[StarlightMerryMakers]GoblinTrust", npc:getID())
-        player:tradeComplete()
-        player:showText(npc, id.text.MERRYMAKER_TRADE)
+    local questStatus = player:getLocalVar("[StarlightMerryMakers]Started")
+    if questStatus ~= 0 then
+        if item == 4495 then
+            player:setLocalVar("[StarlightMerryMakers]GoblinTrust", npc:getID())
+            player:tradeComplete()
```

Comment:
This is fine, but in a follow-up we might want to do:
```
if npcUtil.tradeHasExactly(trade, 4495) then
  player:tradeComplete()
end
```

It's not urgent but the current code allows players to trade extra stuff and the NPC will eat it as long as 4495 is in the first slot